salesforce-marketing-cloud/3.1/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,417 @@
+
+=== Add a Proxy
+
+To use a proxy server, set the following configuration properties on the *Advanced* tab of your configuration:
+
+image::salesforce-mktg-3-proxy.png[Schedule Start Automation]
+
+[[providing-subclass]]
+=== Providing a Subclass as a Type to a Complex Field
+
+Suppose you want to schedule an existing Automation that send emails to a subscriber list once per minute. To do this, add a Schedule Reference to the connector through a flow variable:
+
+image::salesforce-mktg-3-schedule-start-automation.png[Schedule Start Automation]
+
+Use the Recurrence field in ScheduleDefinition to provide information such as how much time should pass between emails sent. The Recurrence field is a complex field that has no structure.
+
+To specify a MinutelyRecurrence instead of a Recurrence:
+
+* Manually add the fields belonging to the MinutelyRecurrence class.
+
+* Add an additional field called concreteClassType of type String whose value is the name of the subclass.
+
+The mapping for the ScheduleDefinition looks like this in the example:
+
+image::salesforce-mktg-3-schedule-definition-transform-config.png[Schedule Definition]
+
+This map has a field called `minuteInterval` that belongs to a subclass of Recurrence called `MinutelyRecurrence`.
+
+For the connector to use the MinutelyRecurrence object, you must also add the concreteClassType field with `MinutelyRecurrence` as the value.
+
+ [[asynchronous-operations]]
+=== Asynchronous Operations
+
+Most operations are synchronous by default, meaning that the connector waits for the result of the operation. For more details regarding the operations of the Marketing Cloud API, access the Salesforce Marketing Cloud Methods documentation.
+
+To specify that you want an operation to behave asynchronously, use the Options parameter for the operation.
+
+The following example creates a list of Automation objects to provide in the payload. Because the result of any operation that works directly with Automation objects throws an exception caused by the presence of an unknown field, the example uses the CreateOptions parameter to make the call asynchronous. In this example, the CreateOptions value is provided in a variable called `vars`.
+
+image::salesforce-mktg-3-create-automation-config.png[Create Automation]
+
+This mapping for CreateOptions in vars looks like this:
+
+* The requestType field determines the type of call (SYNCHRONOUS or ASYNCHRONOUS).
+* The conversationID field assigns a unique identifier to the asynchronous call.
+
+You can group asynchronous calls together using the conversationID, callsInConversation, and sequenceCode fields. For example, suppose you want to make five asynchronous calls to the server, execute the calls together, and specify their execution order. To do this:
+
+* Assign the same conversationID to each call.
```

Comment:
Are these steps or options? It's not clear to me. If they are just diff ways of doing it, then it's fine as an unordered list--otherwise, make it steps.